coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -72,6 +75,23 @@ def parse_python_default(self):
         return self._parse_documentation_with_symbols(lines, param_identifier,
                                                       return_identifier)
 
+    def parse_python_doxygen(self):
+        """
+        Parse python documentation in doxygen style. Usable attributes are:
+
+        - ``@param``
+        - ``@return``
```

Comment:
well doxygen has very less documentation regarding this. Could you point me to a resource?